users/views.py

```diff
@@ -1,3 +1,31 @@
-from django.shortcuts import render
+import json, jwt, requests, random
 
-# Create your views here.
+from django.http  import JsonResponse
+from django.views import View
+
+from users.models import User
+from my_settings  import SECRET_KEY
+
+class KakaoSignInView(View): 
+    def get(self, request): 
+        try:
+            access_token = request.headers.get('Authorization')
+            print(access_token)
+            if not access_token: 
+                return JsonResponse({'message':'no_token'}, status=401)
+
+            url      = 'https://kapi.kakao.com/v2/user/me'
+            response = requests.get(url, headers={'Authorization': f'Bearer {access_token}'})
+            user_response = response.json()
+        
+            user,created = User.objects.get_or_create(kakao_id=user_response['id'])
+            if created: 
```

Comment:
```suggestion
            user,created = User.objects.get_or_create(kakao_id=user_response['id'])
            
            if created: 
```
세로운 로직이 나왔을 때는 공백을 주세요 (ex> if, for 등)

Reply:
수정했습니다!!